### DISCUSSION TITLE
Custom Button to Datatable Row Edit

### DISCUSSION THREAD

**Discussion by mdrakash** [primefaces/community#554] [PrimeVue] [answered]
i want to edit datatable row how to program this. button click to row editable

my custom column this 

`
<Column :rowEditor="true" header="Action" style="width: 10%;">
      <template #body>
          <Button icon="pi pi-check" class="p-button-rounded p-button-primary p-mr-2" />
          <Button icon="pi pi-times" class="p-button-rounded p-button-secondary" />
      </template>
  </Column>
`

**Comment by tugcekucukoglu** [comment, marked as answer]
I want to share a stackblitzlink for this: https://stackblitz.com/edit/v4dxa4?file=src%2FApp.vue%3AL38

**Reply by mdrakash** [reply]
many many thank @tugcekucukoglu

**Reply by ftpereira** [reply]
I need help configuring the dataTable for editing mode. There is a way to let the dataTable start with all fields in edit mode, without having to individually click on the edit button for each row.
